app/assets/javascripts/reducers/course.js

```diff
@@ -29,10 +29,15 @@ const initialState = {
   end: null,
   timeline_start: null,
   timeline_end: null,
+  language: 'en',
+  project: 'wikipedia',
```

Comment:
Why do we need this data duplicated between with the home_wiki?

Reply:
Because in Course Creator when we change the home wiki, we need to add the current home wiki and remove the previous home wiki. In this case previous home wiki would be undefined as language and project didn't have a value initially.